docs/prod-host-setup.md

```diff
@@ -108,7 +124,32 @@ as the Branch Target Injection variant.
 Verification can be done by running:
 
 ```bash
-(grep -Eq '^Mitigation: Full [[:alpha:]]+ retpoline, IBPB: conditional, IBRS_FW' /sys/devices/system/cpu/vulnerabilities/spectre_v2 && echo "retpoline, IBPB, IBRS: ENABLED (OK)") || echo "retpoline, IBPB, IBRS: DISABLED (Recommendation: ENABLED)"
+(grep -Eq '^Mitigation: Full [[:alpha:]]+ retpoline, IBPB: conditional, IBRS_FW' \
+/sys/devices/system/cpu/vulnerabilities/spectre_v2 && \
+echo "retpoline, IBPB, IBRS: ENABLED (OK)") \
+|| echo "retpoline, IBPB, IBRS: DISABLED (Recommendation: ENABLED)"
+```
+
+#### [ARM only] Check that there is mitigation for Spectre Side Channels
```

Comment:
AFAIK this is the same as the previous section (#### [Intel and AMD only] Check for speculative branch prediction issue mitigation) but applied for ARM.
If this is the case, can you please keep the same name.